.github/workflows/ci-aptos-contract.yml

```diff
@@ -19,10 +20,22 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Download CLI
-        run: wget https://github.com/aptos-labs/aptos-core/releases/download/aptos-cli-v3.1.0/aptos-cli-3.1.0-Ubuntu-22.04-x86_64.zip
+        run: wget https://github.com/aptos-labs/aptos-core/releases/download/aptos-cli-v6.1.1/aptos-cli-6.1.1-Ubuntu-22.04-x86_64.zip
 
       - name: Unzip CLI
-        run: unzip aptos-cli-3.1.0-Ubuntu-22.04-x86_64.zip
+        run: unzip aptos-cli-6.1.1-Ubuntu-22.04-x86_64.zip
+
+      - name: Install movefmt
+        run: ./aptos update movefmt
+
+      - name: Check Formatting
+        run: ./aptos move fmt
+        if: success() || failure()
```

Comment:
this if condition runs the job regardless of what happened before, so you get feedback on formatting / lint / tests no matter what.

Reply:
Can we just remove the condition?  Seems redundant...

Reply:
(aside) for posterity: this condition is false when the workflow is cancelled